### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by salceson**
Tests

### CODE REVIEW

File: src/test/scala/pl/edu/agh/iet/akka_tracing/sbt/ConfigParserSpec.scala [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  val fileEmptyPackages = new File(getClass.getResource("/akka_tracing.conf.test_empty").toURI)
+  val fileNoPackages = new File(getClass.getResource("/akka_tracing.conf.test_no").toURI)
+
+  "A ConfigParser" should "return correct package names" in {
+    assert(ConfigParser(fileEmptyPackages).getPackages === List())
+    assert(ConfigParser(file).getPackages ===
+      List("pl.edu.agh.iet.akka_tracing.test.actors1", "pl.edu.agh.iet.akka_tracing.test.actors2"))
+  }
+
+  it should "throw ConfigException.Missing when executed \"getPackages\" with missing \"packages\" key" in {
```
Comment (salceson) [inline src/test/scala/pl/edu/agh/iet/akka_tracing/sbt/ConfigParserSpec.scala]: Use the force(`"""`)

File: src/test/resources/akka_tracing.conf.test [hunk 2]
```diff
@@ -0,0 +1,9 @@
+akka_tracing {
+  packages = [
+    "pl.edu.agh.iet.akka_tracing.test.actors1",
+    "pl.edu.agh.iet.akka_tracing.test.actors2"
+  ]
+  pl.edu.agh.iet.akka_tracing.test.actors1 {
+    actors = ["Actor1"]
+  }
+}
```
Comment (mariusz89016) [inline src/test/resources/akka_tracing.conf.test]: Add newline

File: src/test/resources/akka_tracing.conf.test_empty [hunk 3]
```diff
@@ -0,0 +1,3 @@
+akka_tracing {
+  packages = []
+}
```
Comment (mariusz89016) [inline src/test/resources/akka_tracing.conf.test_empty]: Add newline

File: src/test/resources/akka_tracing.conf.test_no [hunk 4]
```diff
@@ -0,0 +1 @@
+akka_tracing {}
```
Comment (mariusz89016) [inline src/test/resources/akka_tracing.conf.test_no]: Add newline

File: src/test/scala/pl/edu/agh/iet/akka_tracing/sbt/ConfigParserSpec.scala [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+      ConfigParser(fileNoPackages).getPackages
+    }
+  }
+
+  it should "return correct actor names" in {
+    assert(ConfigParser(file).getActors ===
+      List("pl.edu.agh.iet.akka_tracing.test.actors1.Actor1", "pl.edu.agh.iet.akka_tracing.test.actors2.*"))
+  }
+
+  it should "throw ConfigException.Missing when executed \"getActor\" with missing \"packages\" key" in {
```
Comment (mariusz89016) [inline src/test/scala/pl/edu/agh/iet/akka_tracing/sbt/ConfigParserSpec.scala]: use """

File: src/test/scala/pl/edu/agh/iet/akka_tracing/sbt/FilesGeneratorSpec.scala [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    assert(output === new File("./scala/akka/MethodBang.scala"))
+  }
+
+  it should "not generate aspect with the same configuration" in {
+    val output = FilesGenerator.generateAspect(configParser, new File("."), "CONFIG_NAME").head
+    val lines = Source.fromFile(output).mkString
+    val output2 = FilesGenerator.generateAspect(configParser, new File("."), "CONFIG_NAME").head
+    val lines2 = Source.fromFile(output).mkString
+    assert(lines === lines2)
+  }
```
Comment (salceson) [inline src/test/scala/pl/edu/agh/iet/akka_tracing/sbt/FilesGeneratorSpec.scala:24]: Check modification date.

File: src/test/scala/pl/edu/agh/iet/akka_tracing/sbt/FilesGeneratorSpec.scala [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+  it should "generate aspect when configuration changed" in {
+    val output = FilesGenerator.generateAspect(configParser, new File("."), "CONFIG_NAME").head
+    val lines = Source.fromFile(output).mkString
+    val output2 = FilesGenerator.generateAspect(ConfigParser(configFile2), new File("."), "CONFIG_NAME").head
+    val lines2 = Source.fromFile(output).mkString
+    assert(lines !== lines2)
+  }
```
Comment (salceson) [inline src/test/scala/pl/edu/agh/iet/akka_tracing/sbt/FilesGeneratorSpec.scala:61]: Check modification date.

File: src/test/scala/pl/edu/agh/iet/akka_tracing/sbt/ImplicitsSpec.scala [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+import com.typesafe.config.ConfigFactory
+import org.scalatest.FlatSpec
+
+class ImplicitsSpec extends FlatSpec {
+  val configFile = new File(getClass.getResource("/akka_tracing.conf.test").toURI)
+  val config = ConfigFactory.parseFile(configFile)
+
+  import Implicits._
+
+  "A RichConfig" should "return correct values when executing \"getStringListOr\" on Config" in {
```
Comment (mariusz89016) [inline src/test/scala/pl/edu/agh/iet/akka_tracing/sbt/ImplicitsSpec.scala]: use """

File: src/test/scala/pl/edu/agh/iet/akka_tracing/sbt/ImplicitsSpec.scala [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+    assert(config.getStringListOr("akka_tracing.packages", List("")) ===
+      List("pl.edu.agh.iet.akka_tracing.test.actors1", "pl.edu.agh.iet.akka_tracing.test.actors2"))
+  }
+
+  it should "return \"or\" argument when key doesn't exist" in {
```
Comment (mariusz89016) [inline src/test/scala/pl/edu/agh/iet/akka_tracing/sbt/ImplicitsSpec.scala]: use """

**Comment by salceson** [comment]
Ok to merge.

**Pull request merged**